main.tf

```diff
@@ -221,6 +248,30 @@ resource "aws_codebuild_project" "default" {
     location = var.artifact_location
   }
 
+  # Since the output type is restricted to S3 by the provider (this appears to
+  # be an bug in AWS, rather than an architectural decision; see this issue for
+  # discussion: https://github.com/hashicorp/terraform-provider-aws/pull/9652),
+  # this cannot be a CodePipeline output. Otherwise, _all_ of the artifacts
+  # would need to be secondary if there were more than one. For reference, see
+  # https://docs.aws.amazon.com/codepipeline/latest/userguide/action-reference-CodeBuild.html#action-reference-CodeBuild-config.
+  dynamic "secondary_artifacts" {
+    for_each = signum(length(var.secondary_artifact_location)) == 1 ? [""] : []
+    content {
+      type                = "S3"
+      location            = var.secondary_artifact_location
+      artifact_identifier = var.secondary_artifact_identifier
+      encryption_disabled = true
```

Comment:
Can you add an additional variable for `secondary_artifact_encryption_enabled`, set the default to `false`, and use that var here? All Cloud Posse modules are trying to pass BridgeCrew security scanning and I'd bet this might fail in the future if the default / hardcoded value is false.

Reply:
@Gowiem just to clarify, do you want `secondary_artifact_encryption_enabled`, with it set to `false`, or `secondary_artifact_encryption_disabled`, with it set to `false`? I'm guessing you actually meant the latter?

Reply:
I went ahead with that assumption that the variable name should be `secondary_artifact_encryption_disabled`, and set to `false`. I can fix if this is not what you intended.

Reply:
Ah sorry @jhosteny I did mean `secondary_artifact_encryption_enabled`. We have `_enabled` as a convention for any new `bool` vars. 

See https://docs.cloudposse.com/reference/best-practices/terraform-best-practices/#use-positive-variable-names-to-avoid-double-negatives